rotations/demonhunter/vengeance.lua

```diff
@@ -28,30 +24,31 @@ local exeOnLoad = function()
 end
 
 local _All = {
-	-- keybind
-  { "Infernal Strike", "keybind(lalt)" },
-  { "Sigil of Flame", "keybind(lcontrol)" },
-
-	{ "/stopcasting\n/stopattack\n/cleartarget\n/stopattack\n/cleartarget", { "player.combat.time >= 300", (function() return F('dpstest') end) }},
+	-- Keybinds
+	{ "Infernal Strike", "keybind(lalt)" },
+	{ "Sigil of Flame", "keybind(lcontrol)" },
 }
 
 local _Cooldowns = {
-  { "Lifeblood" },
-  { "Berserking" },
-  { "Blood Fury" },
+	--{ "Lifeblood" }, -- No point for this. Only Blood and Night Elves can be Demon Hunters
+	--{ "Berserking" }, -- No point for this. Only Blood and Night Elves can be Demon Hunters
+	--{ "Blood Fury" }, -- No point for this. Only Blood and Night Elves can be Demon Hunters
 }
 
 local _Survival = {
-	{ "Fracture", { "player.pain >= 80", "player.buff(Soul Fragments).count >= 5", "player.pain >= 80" }},
 	{ "Soul Cleave", "player.pain >= 80" },
-	{ "Fel Devastation", "player.health <= 75" },
-	{ "Metamorphosis", { "!player.buff(Demon Spikes)", "!target.debuff(Fiery Brand)", "!player.buff(Metamorphosis)", "player.health <= 70" }},
+	{ "Soul Barrier", { "talent(7,3)", "player.health <= 60" }},
```

Comment:
Got it. Didn't know about that. I think the only line then that might need to have the talent, it's this one:

local _AoE = {
	{ "Immolation Aura", "player.pain <= 80" },
	{ "Sigil of Flame", "talent(6,1)" },
}

In this case, this talents makes the sigils to be cast as an instant cast underneath the player and not as a ground target.